python/generate-pypi-package.sh

```diff
@@ -38,39 +37,29 @@ mkdir -p /tmp/opm/wheelhouse
 
 for tag in ${!python_versions[@]}
 do
-    # Build opm-common bindings
-    pushd opm-common
     # Delete the folder if it already exists
     if [ -d $tag ]; then
       rm -rf $tag
     fi
-    mkdir $tag && cd $tag
-    cmake -DPYTHON_EXECUTABLE=${python_versions[$tag]} -DWITH_NATIVE=0 \
-      -DOPM_ENABLE_PYTHON=ON -DOPM_PYTHON_PACKAGE_VERSION_TAG=${VERSION_TAG} ..
+    mkdir $tag && pushd $tag
+    cmake -DPYTHON_EXECUTABLE=${python_versions[$tag]} -DWITH_NATIVE=0 -DBoost_USE_STATIC_LIBS=1 \
+    -DOPM_ENABLE_PYTHON=ON -DOPM_PYTHON_PACKAGE_VERSION_TAG=${VERSION_TAG} -DBLA_STATIC=1 -DBLAS_LIBRARIES=/usr/lib64/libblas.a -DSUITESPARSE_USE_STATIC=1 -DCMAKE_DISABLE_FIND_PACKAGE_QuadMath=1 ..
+
+    cmake --build . --target opmcommon_python simulators -- -j${BUILD_JOBS}
```

Comment:
If you're changing this anyway, you may as well use the cmake utility's `--parallel` option to write this as
```sh
cmake --build . --target ... --parallel ${BUILD_JOBS}
```
instead.  This will pass the appropriate job option to the underlying build tool instead of assuming that the underlying build tool has a `-j` option.

Reply:
done.